cluster/kubemark/pre-existing/config-default.sh

```diff
@@ -37,3 +37,4 @@ KUBELET_TEST_LOG_LEVEL="${KUBELET_TEST_LOG_LEVEL:-}"
 KUBEPROXY_TEST_LOG_LEVEL="${KUBEPROXY_TEST_LOG_LEVEL:-}"
 MASTER_NAME="${MASTER_NAME:-}"
 USE_REAL_PROXIER="${USE_REAL_PROXIER:-true}"
+ENABLE_KUBEMARK_CLUSTER_AUTOSCALER="${ENABLE_KUBEMARK_CLUSTER_AUTOSCALER:-false}"
```

Comment:
I don't think this variable should be added here (as your master is anyway assumed to be pre-existing). Instead, could you please change [this line](https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/run-e2e-tests.sh#L41) in run-e2e-tests.sh to not fail if the variable is not defined. That is,

```
"${ENABLE_KUBEMARK_CLUSTER_AUTOSCALER:-}"
```